data/json/npcs/isherwood_farm/NPC_Chris_Isherwood.json

```diff
@@ -167,21 +293,381 @@
+        "effect": [ { "u_consume_item": "helmet_riot" }, { "math": [ "isherwood_gear_retrival_helmet_count", "++" ] } ],
+        "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Gear_Turn_Over"
+      },
+      {
+        "text": "[arm armor]",
+        "condition": { "and": [ { "u_has_item": "armor_riot_arm" }, { "math": [ "isherwood_gear_retrival_armor_arm_count", "<", "5" ] } ] },
+        "effect": [ { "u_consume_item": "armor_riot_arm" }, { "math": [ "isherwood_gear_retrival_armor_arm_count", "++" ] } ],
+        "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Gear_Turn_Over"
+      },
+      {
+        "text": "[leg armor]",
+        "condition": { "and": [ { "u_has_item": "armor_riot_leg" }, { "math": [ "isherwood_gear_retrival_armor_leg_count", "<", "5" ] } ] },
+        "effect": [ { "u_consume_item": "armor_riot_leg" }, { "math": [ "isherwood_gear_retrival_armor_leg_count", "++" ] } ],
+        "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Gear_Turn_Over"
+      },
+      {
+        "text": "[got all cop gear]",
+        "condition": {
+          "and": [
+            { "u_has_mission": "MISSION_ISHERWOOD_CHRIS_1_GET_GEAR" },
+            {
+              "and": [
+                { "math": [ "isherwood_gear_retrival_armor_body_count", "==", "5" ] },
+                { "math": [ "isherwood_gear_retrival_helmet_count", "==", "5" ] },
+                { "math": [ "isherwood_gear_retrival_armor_arm_count", "==", "5" ] },
+                { "math": [ "isherwood_gear_retrival_armor_leg_count", "==", "5" ] }
+              ]
+            },
+            { "math": [ "isherwood_gear_status", "<", "1" ] }
+          ]
+        },
+        "effect": { "finish_mission": "MISSION_ISHERWOOD_CHRIS_1_GET_GEAR", "success": true },
+        "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Gear_Done"
+      },
+      { "text": "That's all for now.  Let's talk about something else.", "topic": "TALK_ISHERWOOD_CHRIS_TOPICS" },
+      { "text": "That's all for now.  I'd better get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Gear_Done",
+    "dynamic_line": "[sweet now we beefed up.]",
+    "responses": [
+      { "text": "[woo hoo]  Let's talk about something else.", "topic": "TALK_ISHERWOOD_CHRIS_TOPICS" },
+      { "text": "[woo hoo]", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Gear_Fail",
+    "dynamic_line": "[aw shucks that's bad. now no sweet armor.]",
+    "responses": [
+      { "text": "Sorry.  Let's talk about something else.", "topic": "TALK_ISHERWOOD_CHRIS_TOPICS" },
+      { "text": "[sorry]", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN_Wall",
+    "dynamic_line": "[yeah bro that sounds sweet. I got a plan Dutch, get 20 canvas bags 4 me. I get traps and we lay a wall with da truck]",
```

Comment:
This whole topic needs some dialogue added to it. This line specifically should have Chris plan about the wall.